model/atmosphere/dycore/tests/dycore/mpi_tests/test_parallel_solve_nonhydro.py

```diff
@@ -204,11 +178,6 @@ def test_run_solve_nonhydro_single_step(
         prognostic_states.next.rho.asnumpy(),
     )
 
-    assert test_utils.dallclose(
-        savepoint_nonhydro_exit.rho_ic().asnumpy(),
-        diagnostic_state_nh.rho_ic.asnumpy(),
-    )
-
```

Comment:
I'm missing some context here: why does this not need to be checked?

Reply:
I removed the check because it has been removed from the single node test. Then I assumed it that it is not computed in the dycore anymore, but I didn't check wether this is true.